test/integration/create-next-app/utils.ts

```diff
@@ -0,0 +1,78 @@
+import execa from 'execa'
+import { join } from 'path'
+import { fetchViaHTTP, findPort, killApp, launchApp } from 'next-test-utils'
+
+export const CNA_PATH = require.resolve('create-next-app/dist/index.js')
+export const EXAMPLE_REPO = 'https://github.com/vercel/next.js/tree/canary'
+export const EXAMPLE_PATH = 'examples/basic-css'
+export const FULL_EXAMPLE_PATH = `${EXAMPLE_REPO}/${EXAMPLE_PATH}`
+export const DEFAULT_FILES = [
+  '.gitignore',
+  'package.json',
+  'app/page.tsx',
+  'app/layout.tsx',
+  'node_modules/next',
+]
+
+export const run = (
+  args: string[],
+  options:
+    | execa.Options
+    | {
+        reject?: boolean
+        env?: Record<string, string>
+      }
+) =>
+  execa('node', [CNA_PATH].concat(args), {
+    // tests with options.reject false are expected to exit(1) so don't inherit
+    stdio: options.reject === false ? 'pipe' : 'inherit',
+    ...options,
+    env: { ...process.env, ...options.env },
+  })
+
+export const command = (cmd: string, args: string[]) =>
+  execa(cmd, args, {
+    stdio: 'inherit',
+    env: { ...process.env },
+  })
+
+export async function tryNextDev({
+  cwd,
+  projectName,
+  isApp = true,
+}: {
+  cwd: string
+  projectName: string
+  isApp?: boolean
+}) {
+  const dir = join(cwd, projectName)
+  const port = await findPort()
+  const app = await launchApp(dir, port, {
+    nextBin: join(dir, 'node_modules/next/dist/bin/next'),
```

Comment:
Looks like this might have lost the `turbo` flag to ensure we test both modes

Reply:
Ah I see, yea that seems alright to handle it separately in the turbo test run specifically instead of running both in non-turbo mode